examples/SystemInfo/test-cmd-btrfs-filesystem-show.cc

```diff
@@ -26,7 +26,7 @@ test_cmd_btrfs_filesystem_show(SystemInfo& system_info)
 int
 main()
 {
-    set_logger(get_logfile_logger());
+    Logger::create("/var/log/libstorage-ng.log");
```

Comment:
I don't like the amout of copy&paste that is going on here. I also don't like this vast amount of string literals scattered all over the test suite. This clearly needs to be unified.

Reply:
One of the suggested security improvements was to avoid "hard coded path" for the logs. So we have adapted the Logger to always receive the path of the log file. Anyway, I agree that the testing log path should be defined in only one place. I will fix that.